src/js/media-time-display.js

```diff
@@ -59,12 +66,17 @@ class MediaTimeDisplay extends MediaTextDisplay {
       MediaUIAttributes.MEDIA_CURRENT_TIME,
       MediaUIAttributes.MEDIA_DURATION,
       MediaUIAttributes.MEDIA_SEEKABLE,
-      'remaining',
-      'show-duration',
+      Attributes.REMAINING,
+      Attributes.SHOW_DURATION,
       'disabled',
     ];
   }
 
+  /** @type boolean | undefined */
+  #_showDuration;
+  /** @type boolean | undefined */
+  #_remaining;
```

Comment:
here too the underscores

Reply:
Will either remove or rename before merging, depending on team decision here.